static/styles/components.css

```diff
@@ -144,3 +144,49 @@ select.bootstrap-focus-style {
         outline-offset: -2px;
     }
 }
+
+.progress {
+    overflow: hidden;
+    background-color: hsl(0, 0, 97);
+    background-image: linear-gradient(
+        to bottom,
+        hsl(0, 0%, 96%),
+        hsl(0, 0%, 98%)
+    );
+    background-repeat: repeat-x;
+    box-shadow: inset 0 1px 2px hsla(0, 0%, 0%, 0.1);
+    border-radius: 4px;
+}
+
+.progress .bar {
+    width: 0%;
+    height: 100%;
+    color: hsl(0, 0%, 100%);
+    float: left;
+    font-size: 12px;
+    text-align: center;
```

Comment:
I don't think `text-align` and `font-size` properties are required here since there is no child element for this and we do not have text in it. Can you check this once?

Reply:
And maybe color is also not required.